packages/xo-server/src/xo-mixins/resource-sets.js

```diff
@@ -134,6 +134,19 @@ export default class {
     )
   }
 
+  async computeSnapshotResourcesUsage(snapshot) {
```

Comment:
I'm always afraid it might be unclear that we're talking about VM snapshots.

```suggestion
  async computeVmSnapshotResourcesUsage(snapshot) {
```